modules/component/types/componentProps.ts

```diff
@@ -0,0 +1,23 @@
+import {Action} from '@action-land/core'
+
+/**
+ * Component typing signature
+ * 1. **iState**: Initial state requiered by component to exist.
+ * 2. **oState**: Extends iState may contain additional properties.
+ * 3. **iActions**: Set of actions (messages) handled by component.
+ * 4. **oActions**: Set of actions fired by component for causing side effects.
+ * 5. **oView**: View returned by component.
+ * 6. **iChildren**: Set of child component respresented as key value pair.
+ * 7. **iprops**: Input param types for view.
```

Comment:
iProps

Reply:
the word `types` here is confusing... would recommend just keeping `input params for view` since the idea of it being type-related is already mentioned in the title